velox/expression/tests/CastExprTest.cpp

```diff
@@ -943,6 +943,133 @@ TEST_F(CastExprTest, toString) {
   ASSERT_EQ("cast((a) as ARRAY<VARCHAR>)", exprSet.exprs()[1]->toString());
 }
 
+TEST_F(CastExprTest, doubleToDecimal) {
+  // floating to short decimal
```

Comment:
Nit: please follow the comment style by capitalize the first letter and add `.` at the end.